### ISSUE TITLE
Input Japanese Text in Editor

### ISSUE THREAD

**Issue by spyder-issues-migrator** [spyder-ide/spyder#8]
_From [ruoyu0...@gmail.com](https://code.google.com/u/112768690977086123209/) on 2009-08-24T00:39:51Z_

What steps will reproduce the problem?

 When I input Japanese text into the editor, I can't see what I have inputed 
until I press Enter key. Please see the attach file 
"japanese_input_in_editor.jpg". 

spyder-1.0.0beta2 has the same problem with the Interactive console window,  
but it has been fixed by spyder-1.0.0beta3. Please see the attach file: 
"japanese_input_in_console.jpg". 

So I think maybe the same changes with the console window can fix this problem 
in the editor window. What version of the product are you using? On what operating system? spyder-1.0.0beta3
Japanese Windows XP

**Attachment:** [japanese_input_in_editor.jpg japanese_input_in_console.jpg](http://code.google.com/p/spyderlib/issues/detail?id=8)

_Original issue: http://code.google.com/p/spyderlib/issues/detail?id=8_

**Comment by spyder-issues-migrator** [comment]
_From [pierre.raybaut](https://code.google.com/u/pierre.raybaut/) on 2009-09-22T08:44:57Z_

Thanks for your bug report, and sorry my very late answer.

Unfortunately, the reason why it has been fixed in Interactive console between v1.0.0
beta2 and v1.0.0 beta3 is because I switched from a QScintilla-based console to a
QTextEdit-based (i.e. pure Qt) console. And right now I don't intend to do the same
for the editor widget (too much work because of syntax highlighting implementation in
at least four languages).

So basically, that is the reason why it works for you in the interactive console but
not in the editor... and I don't have any short-term solution, sorry.

However, next year, this kind of work could be done for Spyder v1.1: I'll mention
this in Spyder's roadmap.

**Status:** WontFix
